.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+**Describe the bug**
+Please provide a concise, clear description of the bug, as well as any available error logs.
+
+**Please also include the following items to support reproducing the bug**
+1. compilers (with versions)
+2. Kokkos release or commit used (i.e., the sha1 number)
+3. platform and backend (e.g. Carawy and HIP-AMD, or Weaver and CUDA, OpenMP, etc)
+4. cmake configure command
+5. output from cmake command
+4. command line and / or code needed to reproduce the bug
+5. please also attach the "kokksCore_config.h" header file (generated during the build);
```

Comment:
Can you double-check the name of this file? I believe it's called `KokkosCore_config.h`.